### PR TITLE
Added filter support: by kind, status, both

### PR DESCRIPTION
**Pull request by Megha-Dev-19**

### CODE REVIEW

File: apps/sdks.near/widget/SDKs/Sputnik/DaoSDK.jsx [hunk 1]
```diff
@@ -1,3 +1,17 @@
+const filterFunction = (item, filterStatusArray, filterKindArray) => {
+  const kind = typeof kind === "string" ? kind : Object.keys(item.kind)[0];
```
Comment (matiasberaldo) [inline apps/sdks.near/widget/SDKs/Sputnik/DaoSDK.jsx]: If I'm not mistaken, this ternary operation always falls on the else condition, so maybe a part of the code is missing? Or perhaps it's not needed anymore?

File: apps/sdks.near/widget/SDKs/Sputnik/.prettierrc.json [hunk 2]
```diff
@@ -0,0 +1,6 @@
+{
```
Comment (matiasberaldo) [inline apps/sdks.near/widget/SDKs/Sputnik/.prettierrc.json]: I think this file should be removed

**Review by matiasberaldo — CHANGES_REQUESTED** [review]

File: apps/sdks.near/widget/SDKs/Sputnik/DaoSDK.jsx (hunk 1 above)
Comment (Megha-Dev-19) [reply apps/sdks.near/widget/SDKs/Sputnik/DaoSDK.jsx]: oh yes it should be `item.kind`
thanks for noticing

File: apps/sdks.near/widget/SDKs/Sputnik/.prettierrc.json (hunk 2 above)
Comment (Megha-Dev-19) [reply apps/sdks.near/widget/SDKs/Sputnik/.prettierrc.json]: oops I committed this, will add to .gitignore

**Review by matiasberaldo — APPROVED** [review]

**Pull request merged**